src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainAppTest {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
 }
```

Comment:
no new line at the end of the file

Reply:
as i understand, there is no new line, 11th line with "}" is last in this file so i can`t make new commit, because i can`t change anything

Reply:
I realized my mistake